packages/react-hooks/documentation/Example.tsx

```diff
@@ -1,14 +1,26 @@
-import React, { useState } from "react";
+import React, { useState, useRef } from "react";
 import "@fremtind/jkl-button/button.css";
 import "@fremtind/jkl-core/core.min.css";
+import { useAnimatedHeight, useMutationObserver } from "../src";
 import "./index.scss";
-import { useAnimatedHeight } from "../src";
 
 const Example = () => {
     const [isOpen, setIsOpen] = useState();
+    const [listOfMutations, appendToMutationList] = useState<string[]>([]);
     const [ref] = useAnimatedHeight<HTMLDivElement>(isOpen);
+    const targetRef = useRef(null);
+    const listRef = useRef<string[]>();
+
+    listRef.current = listOfMutations;
+
+    const onObservation = (mutationList: MutationRecord[]) => {
+        appendToMutationList([...(listRef.current || []), ...mutationList.map((item: any) => item.target.data)]);
```

Comment:
Finnes ingen definisjon for `MutationRecord`? Får vel leve med en `any` i så tilfelle.